DemoApp/StreamChat/Components/DemoChatMessageActionsVC.swift

```diff
@@ -82,6 +84,16 @@ final class DemoChatMessageActionsVC: ChatMessageActionsVC {
         )
     }
 
+    func messageDebugActionItem() -> ChatMessageActionItem {
```

Comment:
This is a handy message action. However, not sure if it's best to include it by default, wdyt?

Reply:
This is only for the Demo App. But yes it might be too much actions out of the box 🤔 I'll add a DemoApp config to enable/disable this 👍

Reply:
Done ✅